infinispan/src/main/java/com/yahoo/ycsb/db/InfinispanRemoteClient.java

```diff
@@ -0,0 +1,128 @@
+package com.yahoo.ycsb.db;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.Vector;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.util.logging.Log;
+import org.infinispan.util.logging.LogFactory;
```

Comment:
What is the infinispan logging util package? Does it offer some compelling difference over a more common option like slf4j or commons-logging?